apps/mull-api/src/app/entities/user.entity.ts

```diff
@@ -21,6 +21,10 @@ registerEnumType(RegistrationMethod, {
 @Entity()
 @ObjectType()
 export class User implements IUser {
+  constructor(id?: number) {
```

Comment:
It's not necessary. I added it because it made our code cleaner. without it, to instantiate a user with an id it would take 2 lines like below.
 
The reason I made it optional is because there will be cases where we want to instantiate a new user without an id because the id will be determined by the database. 

```
const user = new User();
user.id = userId
```
vs what I can do now
```
const user = new User(userId)
```

Reply:
👍